APMrover2/mode.cpp

```diff
@@ -66,6 +68,7 @@ void Mode::get_pilot_input(float &steering_out, float &throttle_out)
             throttle_out = rover.channel_throttle->get_control_in();
             const float steering_dir = is_negative(throttle_out) ? -1 : 1;
             steering_out = steering_dir * rover.channel_steer->get_control_in();
+            lateral_out = rover.channel_lateral->get_control_in();
             break;
         }
 
```

Comment:
I think decoding of the lateral value should happen outside of the switch statement so that it applies to all steering types.  It's not clear what a two-paddle input transmitter would look like in order to handle lateral movement but perhaps one of the two sticks would be a 2D stick so the pilot could move the vehicle sideways.

Alternatively we could create a new get_pilot_desired_latera() method.

Reply:
+1